Geometry/TrackerGeometryBuilder/plugins/TrackerDigiGeometryESModule.cc

```diff
@@ -49,14 +85,12 @@ TrackerDigiGeometryESModule::~TrackerDigiGeometryESModule() {}
 void TrackerDigiGeometryESModule::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription descDB;
   descDB.add<std::string>("appendToDataLabel", "");
-  descDB.add<bool>("fromDDD", false);
```

Comment:
that causes a problem:
```
DQMStore: MultiThread option is enabled
----- Begin Fatal Exception 22-Aug-2019 12:44:24 CEST-----------------------
An exception of category 'Configuration' occurred while
   [0] Constructing the EventProcessor
   [1] Validating configuration of ESProducer or ESSource of type TrackerDigiGeometryESModule with label: 'TrackerDigiGeometryESModule'
Exception Message:
Illegal parameter found in configuration.  The parameter is named:
 'fromDDD'
You could be trying to use a parameter name that is not
allowed for this plugin or it could be misspelled.
----- End Fatal Exception -------------------------------------------------
```